dev-libs/pigpio/pigpio-72.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{2_7,3_{5,6,7}} )
```

Comment:
Please look into deprecating python2.7 support, and into adding python3.8 for next version release.